linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_renameat2/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
@yarunachalam I was advised on one of my changes that since this is SLE only, we should not have ocp4 tests,
but since all uses of complete_ocil_entry_audit_syscall I saw did this, I am not sure what to suggest. On https://github.com/ComplianceAsCode/content/pull/6802 we have simple pass and fail tests, but that change has not landed yet. So, I am not sure that it is the correct approach.

Reply:
Correct, since the prodtype is only sle15, no ocp4 tests are needed. You can see example of tests on this rule, which uses the same template:
https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/tests
Note that the ocp4 folder is still there, but the "prodtype" contains rhcos4. So here it makes sense.

Reply:
@vojtapolasek thank you, you answered a basic question I had, the ocp4 stuff is rhcos4, could not get a good handle on it and thought it might just might be something like open scap version 4.

@yarunachalam please remove  ocp4 files.